public/app/plugins/panel/table/transformers.ts

```diff
@@ -73,6 +73,56 @@ transformers['timeseries_to_columns'] = {
   }
 };
 
+transformers['grouping_to_rows_columns'] = {
+  description: 'Grouping to rows and columns',
+  getColumns: function() {
+    return [];
+  },
+  transform: function(data, panel, model) {
+    // Getting the columns
+    const points = new Map<string, Map<string, any>>();
```

Comment:
All the following code is considering that two groupings have been made. That means, that doing more groupings will show wrong data. I can transform this to consider a multidimensional data structure, but, does it make sense? I can represent the first dimension as columns, the second dimension as rows and then, denormalize the data doing some kind of array representation ( ``[[a,b],[[c,d]]``)